src/options/options.html

```diff
@@ -54,60 +54,81 @@
-					This is useful on websites that do not support changing the font and/or text formatting.</span>
+					<h1 data-i18n="__MSG_titleContextMenu__">Context menu</h1>
+					<span class="helper-text" data-i18n="__MSG_optionHelperTextContextMenu__">Write, select text and choose the corresponding option in the context menu to quickly change the character style or capitalization.
+					This is useful on websites that do not support changing the font or text formatting.</span>
 
 					<li>
 						<div class="line">
 							<input class="setting save-on-change" type="checkbox" id="changeFont" data-optiongroup="unicodeFont" name="changeFont">
-							<label data-i18n="" for="changeFont">Show Unicode font options</label>
+							<label data-i18n="__MSG_optionMenuUnicodeFont__" for="changeFont">Show Unicode font options</label>
 						</div>
-						<span data-i18n="" class="line indent helper-text">For example, ˢᵘᵖᵉʳˢᶜʳⁱᵖᵗ, ꜱᴍᴀʟʟ ᴄᴀᴘꜱ, 𝒮𝒸𝓇𝒾𝓅𝓉, 𝔉𝔯𝔞𝔨𝔱𝔲𝔯, 🅒🅘🅡🅒🅛🅔🅓 and 🆂🆀🆄🅰🆁🅴🅳.</span>
+						<span data-i18n="__MSG_optionMenuUnicodeFontDescr__" class="line indent helper-text">For example, ˢᵘᵖᵉʳˢᶜʳⁱᵖᵗ, ꜱᴍᴀʟʟ ᴄᴀᴘꜱ, 𝒮𝒸𝓇𝒾𝓅𝓉, 𝔉𝔯𝔞𝔨𝔱𝔲𝔯, 🅒🅘🅡🅒🅛🅔🅓 and 🆂🆀🆄🅰🆁🅴🅳.</span>
 					</li>
 
 					<li>
 						<div class="line">
 							<input class="setting save-on-change" type="checkbox" id="changeCase" data-optiongroup="unicodeFont" name="changeCase">
-							<label data-i18n="" for="changeCase">Show case changing options</label>
+							<label data-i18n="__MSG_optionMenuCasing__" for="changeCase">Show case changing options</label>
 						</div>
-						<span data-i18n="" class="line indent helper-text">For example, “UPPER CASE”, “lower case”, “Capitalize Each Word” or “tOGGLE cASE”.</span>
+						<span data-i18n="__MSG_optionMenuCasingDescr__" class="line indent helper-text">For example, “UPPER CASE”, “lower case”, “Capitalize Each Word” or “tOGGLE cASE”.</span>
+					</li>
+
+					<h2 data-i18n="__MSG_subtitleEntryPreview__">Entry design</h2>
+					<li>
+						<div class="line">
+							<input class="setting save-on-change" type="checkbox" id="showReadableText" data-optiongroup="unicodeFont" name="showReadableText">
+							<label data-i18n="__MSG_optionMenuShowReadableText__" for="showReadableText">Show a readable description of the transformation</label>
+						</div>
+						<span data-i18n="__MSG_optionMenuShowReadableTextDescr__" class="line indent helper-text">In addition to the examples of the transformation, this also shows the name of transformation as text.</span>
+					</li>
+
+					<li>
+						<div class="line">
+							<input class="setting save-on-change" type="checkbox" id="livePreview" data-optiongroup="unicodeFont" name="livePreview">
+							<label data-i18n="__MSG_optionMenuShowLivePreview__" for="livePreview">Show a live preview with the selected text</label>
+						</div>
+						<span data-i18n="__MSG_optionMenuShowLivePreviewDescr__" class="line indent helper-text">Shows the effect of the transformation on the selected text instead of an example one.</span>
```

Comment:
Okay, for now I've adjusted the `AutomaticSettings` to just handle it the same as mobile-incompatible settings and hide them. May be changed later though.

Reply:
Your change will hide the option for Thunderbird, in addition to Chrome. This line:
https://github.com/TinyWebEx/AutomaticSettings/blob/f67aef00fed936456f009f29a7518d6b38056c78/EnvironmentalOptions.js#L36
would need to be:
```js
return browserInfo.name === "Firefox" || browserInfo.name === "Thunderbird";
```
although maybe also rename the function to something like `isMozilla` or `isGecko`.

Reply:
The vendor name does not work on Thunderbird, since it is blank: 
![image](https://user-images.githubusercontent.com/15305150/112818434-60b56100-9038-11eb-8392-4531d12cb44b.png)

We should either submit a bug to them or change your library to use the name instead.